### PR TITLE
[master] Jakarta dependencies upgrade from EE 9.0.1

### PR DESCRIPTION
**Pull request by rfelcman**
Most dependencies are in staging, some of them (JAXB Impl) are in progress now.
Signed-off-by: Radek Felcman <radek.felcman@oracle.com>

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -162,21 +162,21 @@
...
         <annotation.version>2.0.0</annotation.version>
         <cdi.version>3.0.0</cdi.version>
         <ejb.version>4.0.0</ejb.version>
         <el.version>4.0.1</el.version>
         <glassfish.version>6.0.0</glassfish.version>
         <jaxwsrt.version>3.0.0</jaxwsrt.version>
         <jaxb.version>3.0.0</jaxb.version>
-        <jaxb.api.version>3.0.0</jaxb.api.version>
+        <jaxb.api.version>3.0.1</jaxb.api.version>
         <soap.api.version>2.0.0</soap.api.version>
```
Comment (lukasj) [inline pom.xml:174]: there is 2.0.1 for soap-api as well

**Comment by lukasj** [comment]
for jaxb - there is 3.0.1-b02 in staging now

**Comment by rfelcman** [comment]
>  3.0.1-b02

OK this version is updated too.

**Review by lukasj — APPROVED** [review]
update JAXB-Impl to 3.0.1 once it's available in staging and feel free to merge afterwards

**Pull request merged**